Sources/SKCore/Toolchain.swift

```diff
@@ -73,6 +73,12 @@ public final class Toolchain {
   }
 }
 
+extension Toolchain: Equatable {
+  public static func ==(lhs: Toolchain, rhs: Toolchain) -> Bool {
+    return lhs.identifier == rhs.identifier && lhs.path == rhs.path
```

Comment:
This is not a reasonable `Equatable` conformance; it's only valid to unique by the path/identifier in the context of the registry, not of individual toolchains.  Using a more explicit `contains(where:)` is better.

Reply:
Done